unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatWindowController.cs

```diff
@@ -206,9 +205,6 @@ private void HandleMessageReceived(ChatMessage message)
     
     private async UniTaskVoid WaitThenFadeOutMessages(CancellationToken cancellationToken)
     {
-        if (notificationPanelTransform.Get() == null)
```

Comment:
is this method still useful now that the delay has been removed?

Reply:
I left it in the structure because it could be useful when adding the whole system of animations in the future but I could as well remove it